egs/malach/s5/README.txt

```diff
@@ -0,0 +1,165 @@
+
+This s5 recipe for MALACH data is a modified version of the s5b
+recipe for AMI. 
+
+You need to download the malach data to get started. For information about the MALACH database see : 
+USC-SFI MALACH Interviews and Transcripts English - Speech Recognition Edition
+https://catalog.ldc. upenn.edu/LDC2019S11
+
+Once the data is unloaded and untar-ed, you need to run:
+
+run_prepare_shared.sh - prepares most of the data for the system
+run.sh - builds the system
+
+Beforehand, you need to edit BOTH scripts to point to 
+where you downloaded and untar-ed the data. Find the lines in
+run_prepare_shared.sh and run.sh that say:
+
+malach_dir=dummy_directory
+
+Replace "dummy_directory" with the fully-qualified location of the actual data
+data. For example, let's say you copied the data distribution tar file to 
+/user/jdoe/malach and untar-ed it there. That would create a high level directory called 
+/user/jdoe/malach/malach_eng_speech/recognition. You would then change the above line to read:
+
+malach_dir=/user/doe/malach/malach_eng_speech_recognition/data
+
+Note that the scripts were "tweaked" to always use sclite scoring
+(vs. default kaldi scoring).
+
+Other issues that we have run up against in setting up this recipe
+that may or may not impact you:
+
+On the system on which these scripts were developed, we run python 2.7
+and a relatively older version of CUDA by default. We had to modify
+path.sh to point to the right load libraries for both python 3 (a
+number of the scripts use python three) and an appropriate library
+consistent with the level of CUDA we were using. Please modify path.sh
+accordingly.
+
+You may also have to modify "configure" line 405 in
+/speech7/picheny5_nb/forked_kaldi/kaldi/src to point to where your
+version of CUDA lives. 
+
+Basic pipeline results summary:
+
+tri2:
+
+ascore_7 /dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 64.8   29.4    5.8    9.3   44.6   81.6 | -0.765 |
+ascore_8 /dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 65.7   28.2    6.2    8.5   42.8   79.6 | -0.589 |
+ascore_9 /dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 66.2   27.3    6.5    7.7   41.5   77.9 | -0.451 |
+ascore_10/dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 66.6   26.4    7.0    7.0   40.4   76.7 | -0.351 |
+ascore_11/dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 66.9   25.7    7.4    6.6   39.7   76.0 | -0.293 |
+ascore_12/dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 66.8   25.4    7.8    6.2   39.3   75.0 | -0.257 |
+ascore_13/dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 66.5   25.1    8.3    5.7   39.1   74.0 | -0.230 |
+ascore_14/dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 66.0   25.0    9.0    5.5   39.4   74.3 | -0.231 |
+ascore_15/dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 65.6   24.8    9.6    5.2   39.6   74.0 | -0.218 |
+
+
+tri3.si:
+ascore_7 /dev.ctm.filt.sys: | Sum/Avg  |  843   12345 | 63.0   30.5    6.6    8.5   45.6   79.8 | -1.738 |
```

Comment:
I don't like showing all the different scorings here, as it's overwhelming.  You can do something like:
```
 for dir in exp/tri*/decode*; do    grep Sum $dir/ascore*/*.sys | utils/best_wer.pl;  done
```

Reply:
Done. Have a look.